src/utils.js

```diff
@@ -35,3 +37,70 @@ export function buildDefinition( modelAttributeKey, options ) {
 
 	return definition;
 }
+
+export const FONT_COLOR = 'fontColor';
+export const FONT_BACKGROUND_COLOR = 'fontBackgroundColor';
+
+export function renderUpcastAttribute( styleAttr ) {
+	return viewElement => {
+		const fontColor = viewElement.getStyle( styleAttr );
+		return normalizeColorCode( fontColor );
+	};
+}
+
+export function renderDowncastElement( styleAttr ) {
+	return ( modelAttributeValue, viewWriter ) => viewWriter.createAttributeElement( 'span', {
+		style: `${ styleAttr }:${ modelAttributeValue }`
+	} );
+}
+
+function normalizeColorCode( value ) {
+	return value.replace( /\s/g, '' );
+}
+
+export function normalizeOptions( colorRow ) {
+	return colorRow
+		.map( getColorsDefinition )
+		.filter( option => !!option );
+}
+
+function getColorsDefinition( color ) {
+	if ( typeof color === 'string' ) {
+		return {
+			model: color,
+			label: color,
+			view: {
+				name: 'span',
+				styles: {
+					color
+				},
+				priority: 5
+			}
+		};
+	} else {
+		return {
+			model: color.color,
+			label: color.label,
+			hasBorder: color.hasBorder,
+			view: {
+				name: 'span',
+				styles: {
+					color: `${ color.color }`
+				},
+				priority: 5
+			}
+		};
+	}
+}
+
+export const colorUI = {
```

Comment:
As I wrote earlier, this namespace is pointless with a single function like this.